chain/epoch_manager/src/lib.rs

```diff
@@ -142,6 +142,62 @@ impl EpochManager {
         Ok(epoch_manager)
     }
 
+    /// Only used in mock network
```

Comment:
if that is the case it is better to guard this function with some feature flag

Reply:
Done